modules/CircuitCalculator.py

```diff
@@ -130,7 +122,7 @@ def _generateNoiseFrequencySpec(self, freq, **kwargs):
         '''
         # load info
         Rdc                 = kwargs.get('Rdc', 1) # in Om
-        Rslope              = kwargs.get('Rslope', 1.) # in 1/MHz
+        Rslope              = Rslope_dict(np.abs(freq))
```

Comment:
where is the parameters for Rac now?

Reply:
why you need a dedicated dictionary to input? why not use existing **kwargs?

Reply:
> where is the parameters for Rac now?
there are two noises,and their Rac are different.
in Changqing's file,  Rac is determined by two parameter a and b, but for different noises,a and b has different meaning(different function) , i use interpolation to fit that function.

Reply:
> why you need a dedicated dictionary to input? why not use existing **kwargs?

there will be a bug if using **kwargs cause i do not know how to fix it(or simply i do not really understand how to use **kwargs), so i just add a parameter as input